rtrans-esb/src/main/java/ca/bc/gov/moh/rtrans/service/version/PopulateVersionInformation.java

```diff
@@ -0,0 +1,55 @@
+package ca.bc.gov.moh.rtrans.service.version;
+
+import java.util.Optional;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ca.bc.gov.moh.rtrans.RTransMainMethod;
+import net.minidev.json.JSONObject;
+
+/**
+ * This class is created to get the manifest information and provide the same in version route as JSON object.
+ * Manifest information is set in pom.xml 
+ */
+public class PopulateVersionInformation implements Processor {
+	private static final Logger logger = LoggerFactory.getLogger(PopulateVersionInformation.class);
+	private static final String IMPLEMENTATION_VERSION_KEY = "Implementation-Version";
+	private static final String VERSION_INFORMATION = getVersionInformation().toJSONString();
+
+	/**
+	 * Loading the version information using Package class.
+	 * Version information is set in jar's META-INF/Manifest.mf file
+	 * @return JSONObject
+	 */
+	public static JSONObject getVersionInformation() {
+		Package pck = RTransMainMethod.class.getPackage();
+		final String methodName = getMethodName();
```

Comment:
We don't need to capture the methodName since this isn't done elsewhere in RTrans.